### ISSUE TITLE
[4.0] Draggable

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#21787]
The following admin views have not been converted to use draggable for sorting so the sorting is currently broken

```
index.php?option=com_languages&view=languages
index.php?option=com_users&view=levels
index.php?option=com_content&view=featured
```

However before fixing I can't see the value/use case in sorting for language and levels - am I missing something or is it not needed

[UPDATED]
It appears even where we have draggable that we have an issue saving the ordering with a 500 error
`administrator/index.php?option=com_content&task=articles.saveOrderAjax&tmpl=component
`

**Comment by brianteeman** [comment]
Updated original post

**Comment by brianteeman** [comment]
closed see #22049
